spec/features/access_permission_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+describe "access permissions of pages" do 
```

Comment:
Unless I'm reading this wrong, this won't catch a regular user (lowest clearance) who *is* able to access app configuration or first_run page?